Resources/Prototypes/Catalog/Fills/Boxes/service.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  name: cleanades box
+  parent: BoxCardboard
+  id: BoxCleanerGrenades
+  description: A box full of cleanades.
+  components:
+  - type: StorageFill
+    contents:
+      - id: CleanerGrenade
+        amount: 4
+  - type: Sprite
+    layers:
+      - state: box
+      - state: flashbang
```

Comment:
```suggestion
      - state: flashbang

```

Reply:
can be merged after this

Reply:
This is here for the Box Icon
![image](https://github.com/Simple-Station/Einstein-Engines/assets/45297731/adea5b91-4889-4ab9-9a9a-0477ce9bd78c)

Reply:
yuh but i like having a line at the end of the file, psychological thing and Death instilled it into me